docs/key-features/features.md

```diff
@@ -8,7 +8,7 @@ Karmada supports:
 
 * Safe isolation:
   * Create a namespace for each cluster, prefixed with `karmada-es-`.
-* [Multi-mode](https://karmada.io/docs/userguide/clustermanager/cluster-registration) connection:
```

Comment:
Seems this link is still valid, can you remind me what's the benefit of using a relative path?

Reply:
It will automatically switch to the file of the current version with the change of versions.

Please refer to: https://github.com/karmada-io/website/pull/516#discussion_r1401477114